cli/cmd/plugin/unmanaged-cluster/cmd/create.go

```diff
@@ -63,7 +64,8 @@ func init() {
 	CreateCmd.Flags().StringSliceVarP(&co.portMapping, "port-map", "p", []string{}, "Ports to map between container node and the host (format: '80:80/tcp' or just '80')")
 	CreateCmd.Flags().Bool("tty-disable", false, "Disable log stylization and emojis")
 	CreateCmd.Flags().BoolVar(&co.skipPreflightChecks, "skip-preflight", false, "Skip the preflight checks; default is false")
-	CreateCmd.Flags().StringVarP(&co.numNodes, "nodes", "n", "", "The number of nodes to deploy; default is 1 control-plane")
+	CreateCmd.Flags().StringVar(&co.numContPlanes, "control-plane-node-count", "", "The number of control plane nodes to deploy; default is 1. NOTE: Multiple control-plane nodes without a worker node is not supported")
```

Comment:
nit: I think we should remove `NOTE: Multiple control-plane nodes without a worker node is not supported`, namely because this is a decision for the provider implementation take make.

Reply:
Maybe add "when using the 'kind' provider? It seems like we should have some mention of it if we are going to reject the request later on.

Or we could just set worker node count to 1 if not specified when control plane node count is more than 1.

Reply:
Very true - great catch Josh!

Reply:
Whoops - didn't refresh to see your comment Sean. Went ahead and removed it.

Setting the control-plane node to 1 regardless of the users input feels weird to me. But we should have something that communicates what's happening.

Here's an alternative approach I've considered as well:

```shell
# Create a cluster with 2 control plane nodes
# Installation will stale (not error out) and print warning message that user needs to remove taints manually
$ 

...

🚀 Creating cluster test
   Base image downloaded
   Cluster created
   To troubleshoot, use:

   kubectl ${COMMAND} --kubeconfig /home/jmcb/.config/tanzu/tkg/unmanaged/test/kube.conf

⚠️ Warning! Kind based clusters with multiple control planes 
   and no worker nodes will NOT have workloads scheduled to them
   You must remove taints manually in order for installation to continue:

   kubectl --kubeconfig ~/.config/tanzu/tkg/unmanaged/test/kube.conf taint nodes --all node-role.kubernetes.io/master-
   
📧 Installing kapp-controller
   kapp-controller status: Pending...

----

# In a separate terminal, a user can remove the taints using the following command
$ ❯ kubectl --kubeconfig ~/.config/tanzu/tkg/unmanaged/test/kube.conf taint nodes --all node role.kubernetes.io/master-
node/test-control-plane untainted
node/test-control-plane2 untainted

-----

# Installation resumes now that kapp controller can be scheduled onto the control planes
📧 Installing kapp-controller
   kapp-controller status: Running

📧 Installing package repositories
   Core package repo status: Reconcile succeeded

...
```

This workflow worked locally after removing the block that errors if there are no worker nodes to schedule to.

Reply:
I'm happy to revisit approaches outside this PR.

However, for now, I believe surfacing whether control plane / worker values are valid should be done in the cluster package, via providers. It's something providers can also surface in their pre-flight checks. I'm not too crazy about having conditionals in our flag help text.